bokeh/models/sources.py

```diff
@@ -210,6 +210,18 @@ class RemoteSource(DataSource):
 class AjaxDataSource(RemoteSource):
     method = String('POST', help="http method - GET or POST")
 
+    overwrite = Bool(True, help="""
+    Overwrite or Update plot values from data source object
+    """)
+    max_size = Int(help="""
+    Maximum size that a plot could expand when data is retreive during
+    polling updates. Larger than that size, the data will shift to the right
+    """)
+    if_modified = Bool(False, help="""
+    Polling requests to the server will provide a If-Modified-Since header
+    in order to return only new data to the plot
```

Comment:
Suggest:
```
Whether to include an \`\`If-Modified-Since\`\` header in the AJAX requests to the server. 
```
I am a big fan of consistent terminology. In general for boolean options I try to use "*Whether* to do this (if true) or not."